### ISSUE TITLE
Typeahead issue

### ISSUE THREAD

**Issue by sahlouls** [valor-software/ngx-bootstrap#1135]
Using  Angular 2.1.0 with Webpack

Imported TypeaheadModule 

import { TypeaheadModule } from 'ng2-bootstrap/ng2-bootstrap';

When using

```
         <input
            *ngIf="applications"
            [typeahead]="applications"
            (typeaheadOnSelect)="typeaheadOnSelect($event)"
            class="form-control"
            placeholder="Select Application">
```

Having issue

```
Can't bind to 'typeahead' since it isn't a known property of 'input'. ("
          <input
            *ngIf="applications"
            [ERROR ->][typeahead]="applications"
            (typeaheadOnSelect)="typeaheadOnSelect($event)"
```

**Comment by sahlouls** [comment]
Solved.

**Comment by sdornan** [comment]
What was the issue?

**Comment by critchie** [comment]
I managed to fix this by adding additional imports in the modules that actually use the typeahead.  It wasn't enough to add it the Typeahead.forRoot() in the app module.  In a module that actually uses the typeahead directive I had to add an import statement `import { TypeaheadModule } from 'ng2-bootstrap/typeahead';` and a line in my imports array `imports: [..., TypeaheadModule, ...]`  I had to do the same thing for a module that is using the tabs directive.  Either I'm doing something wrong or the documentation needs to be more clear on this "missing" step.
